docs/content/patterns/alz/UpdateToNewReleases/Update_from_release_2024-03-01.md

```diff
@@ -28,20 +28,20 @@ To run the script, complete the following step:
 
   {{% include "PowerShell-ExecutionPolicy.md" %}}
 
-  **Generate a list of the resource IDs which would be deleted by this script:**
+  **Show output of what would happen if deletes executed:**
 
   ```powershell
-  ./Remove-AMBANotificationAssets.ps1 -pseudoRootManagementGroup $pseudoRootManagementGroup -ReportOnly
+  ./Remove-AMBANotificationAssets.ps1 -pseudoManagementGroup $pseudoRootManagementGroup -WhatIf
```

Comment:
Can you validate? Looks like the param of the script is `[string]$pseudoRootManagementGroup`

Reply:
Done